examples/step-35/step-35.cc

```diff
@@ -508,35 +521,35 @@ namespace Step35
-    // for these two kinds of variables. We pay for this optimization when we
-    // want to assemble terms that involve both variables, such as the
-    // divergence of the velocity and the gradient of the pressure, times the
-    // respective test functions. When doing so, we can't just anymore use a
-    // single FEValues object, but rather we need two, and they need to be
-    // initialized with cell iterators that point to the same cell in the
-    // triangulation but different DoFHandlers.
+    // One of the things that are specific to this program is that we don't just
+    // have a single DoFHandler object that represents both the velocities and
+    // the pressure, but we use individual DoFHandler objects for these two
+    // kinds of variables. We pay for this optimization when we want to assemble
+    // terms that involve both variables, such as the divergence of the velocity
+    // and the gradient of the pressure, times the respective test functions.
+    // When doing so, we can't just anymore use a single FEValues object, but
+    // rather we need two, and they need to be initialized with cell iterators
+    // that point to the same cell in the triangulation but different
+    // DoFHandlers.
     //
-    // To do this in practice, we declare a "synchronous" iterator -- an
-    // object that internally consists of several (in our case two) iterators,
-    // and each time the synchronous iteration is moved forward one step, each
-    // of the iterators stored internally is moved forward one step as well,
-    // thereby always staying in sync. As it so happens, there is a deal.II
-    // class that facilitates this sort of thing. (What is important here is to
-    // know that two DoFHandler objects built on the same triangulation will
-    // walk over the cells of the triangulation in the same order.)
-    typedef std::tuple<typename DoFHandler<dim>::active_cell_iterator,
-                       typename DoFHandler<dim>::active_cell_iterator>
-      IteratorTuple;
-
-    typedef SynchronousIterators<IteratorTuple> IteratorPair;
+    // To do this in practice, we declare a "synchronous" iterator -- an object
+    // that internally consists of several (in our case two) iterators, and each
+    // time the synchronous iteration is moved forward one step, each of the
+    // iterators stored internally is moved forward one step as well, thereby
+    // always staying in sync. As it so happens, there is a deal.II class that
+    // facilitates this sort of thing. (What is important here is to know that
+    // two DoFHandler objects built on the same triangulation will walk over the
+    // cells of the triangulation in the same order.)
+    using IteratorTuple =
+      std::tuple<typename DoFHandler<dim>::active_cell_iterator,
+                 typename DoFHandler<dim>::active_cell_iterator>;
+
+    using IteratorPair = SynchronousIterators<IteratorTuple>;
```

Comment:
We should probably decide (in the style guide) whether or not we want to use the new `using` syntax everywhere or use `typedef`. We should probably bless one or the other.

I have a slight preference for `typedef` out of habit (though, of course, `using` supports more things).